lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -38,9 +38,9 @@ def initialize(app, options = {})
           @options = options
         end
 
-        def configure
+        def configure(reload = false)
```

Comment:
mostly a problem with the current concept of configuration; with the new `Configuration` it should be easy to update the Rack middleware configurations without reinitializing from scratch Rails from our tests. Consider that this `reload` parameter is here only for testing purposes (and it's not ideal).